Orchid.Jint/JsExecutionContext.cs

```diff
@@ -92,14 +94,14 @@ public void RunScript(string name, string script)
         {
             try
             {
-                _engine.Execute(script, new ParserOptions
+              _engine.Execute(script, new ParserOptions
                 {
-                    Source = name
+                  Source = name
                 });
-            }
+              }
             catch (JavaScriptException jsError)
             {
-                Log.Warning("Scripting", "[{0}:{1}] {2}", name, jsError.LineNumber, jsError.Message);
+                Log.Warning("Scripting", "[{0}:{1}] {2}", name, jsError.Location.Start.Line, jsError.Message);
```

Comment:
should we use string interpolation here?

Reply:
I'm starting to think that string interpolation for logging is not the right approach because it doesn't work well with localization. Let's create a new best practice before starting to change code in a way we might need to undo.